### PR TITLE
Michaelbranch

### PR DESCRIPTION
**Pull request by mmtolsma**
Testing out a pull request. Changes compared to Master are minor since I already made changes to the "add-loading-bar" branch and merged like a chop. The changes you made were 99% solid, barring a couple syntax niggles.

### CODE REVIEW

File: lib/app_screens/secondpage.dart [hunk 1]
```diff
@@ -63,7 +63,7 @@ class SecondPageState extends State<SecondPage> {
   }
 
   Widget loadingBar() {
-    return Text("Finding specials...", textAlign: TextAlign.center,);
+    return Text("Fing specials...", textAlign: TextAlign.center,);
```
Comment (ponelat) [inline lib/app_screens/secondpage.dart]: Typo

File: lib/app_screens/secondpage.dart [hunk 2]
```diff
@@ -44,26 +44,28 @@ class SecondPageState extends State<SecondPage> {
 
   @override
   Widget build(BuildContext context) {
-
     var secondPageBody;
 
-    if(data == null) {
+    if (data == null) {
```
Comment (mmtolsma) [inline lib/app_screens/secondpage.dart]: Not sure where that came from. Will fix

File: lib/app_screens/secondpage.dart [hunk 3]
```diff
@@ -44,26 +44,28 @@ class SecondPageState extends State<SecondPage> {
...
-        appBar: AppBar(
-          title: Text("Star Wars Species"),
-          backgroundColor: Colors.green,
-        ),
-        body: secondPageBody,
+      appBar: AppBar(
+        title: Text("Star Wars Species"),
+        backgroundColor: Colors.green,
+      ),
+      body: secondPageBody,
```
Comment (mmtolsma) [inline lib/app_screens/secondpage.dart]: Alignment fix

File: lib/app_screens/secondpage.dart [hunk 4]
```diff
@@ -76,22 +78,33 @@ class SecondPageState extends State<SecondPage> {
...
-                ],
-            ))
+            Image.asset(
+              'images/boo_actual_character.png',
+              height: 80.0,
+              width: 80.0,
+              alignment: Alignment.centerLeft,
+            ),
+            ListTile(title: Text("Language: $language")),
+            //Text(data[index]['language']),
```
Comment (ponelat) [inline lib/app_screens/secondpage.dart]: No need to comment code... with git, we have a history. So either keep the code (uncomment) or remove entirely.
Note: this is an advanced suggestion (one I'd give my colleagues).

Comment (mmtolsma) [reply lib/app_screens/secondpage.dart]: Thanks. Just made one more change for the day. Nothing ground breaking. Will try keep the spamming to a minimum from now. Been trying to get a hang of the "how-to".

Comment (ponelat) [reply lib/app_screens/secondpage.dart]: no problem. Its not spamming, its easy to ignore. If you really require something a ping in whatsapp will have to do. I can ignore GitHub pings

**Comment by ponelat** [comment]
@mmtolsma don't forget to delete the branch

**Pull request merged**